src/components/Menu/__tests__/auth-menu.test.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-env jest */
+import React from 'react';
+import { mount } from 'enzyme';
+import { BrowserRouter } from 'react-router-dom';
+import AuthMenuItem from '../AuthMenuItem';
+
+it('renders Menu without authentication component without crashing', () => {
+  const wrapper = mount(
+    <BrowserRouter>
+      <AuthMenuItem authenticated={false} />
+    </BrowserRouter>,
+  );
+  expect(wrapper.find('#login')).toHaveLength(1);
+});
+
+it('renders AuthMenuItem authentication component without crashing', () => {
+  const wrapper = mount(
+    <BrowserRouter>
```

Comment:
you can also have a look at [Browser router context](https://www.npmjs.com/package/react-router-enzyme-context) as it solves this problem too. though it's optional, if rapping the component with browser router is working fine.

Reply:
This is really helpful. Thank you